Documentation/source/Integration.md

```diff
@@ -0,0 +1,57 @@
+Integration
+=============================
+
+As mentioned before the Gini Vision Library provides two integration options. A [Screen API](#screen-api) that is easy to implement and a more complex, but also more flexible [Component API](#component-api). Both APIs can access the complete functionality of the library.
```

Comment:
I think just stating that GVL provides two integration options is sufficient. "As mentioned before" confused me since I don't know to which section _before_ refers to...